### ISSUE TITLE
My Profile/Edit User window close doesn't prompt for unsaved changes 

### ISSUE THREAD

**Issue by cseppan** [USEPA-OAQPS/emf#61]
Steps to reproduce:

From the Manage menu, select My Profile
Make some kind of change to the user info
Click the window close button (upper right hand corner). The "Discard changes?" confirmation should be displayed but isn't.
